mod/metrics/lib/formula/wiki_rating.rb

```diff
@@ -2,8 +2,10 @@ module Formula
   class WikiRating < Translation
     def get_value input, _company, _year
       result = 0.0
+      total_weight = 0
       input.each.with_index do |value, index|
         weight = @executed_lambda[@input.card_id(index)]
+        total_weight += weight
         result += value.to_f * weight
       end
       result / 100
```

Comment:
unless the result is divided by the total weight (instead of 100), this won't change anything